dart/lib/src/protocol/sentry_runtime.dart

```diff
@@ -40,12 +41,16 @@ class SentryRuntime {
   /// and version from this string, if they are not explicitly given.
   final String? rawDescription;
 
+  /// Application build string, if it is separate from the version.
+  final String? build;
```

Comment:
Maybe we set from `Platform.version`? Right now its `rawDescription` but not sure if it makes sense still

Reply:
Good point. `Platform.version` results in something like `2.18.1 (stable) (Tue Sep 13 11:42:55 2022 +0200) on "ios_arm64"`, so it makes sense to use the `build` field. Out of `Platform.version` the name of the runtime can't be inferred anyway, so it has to be set manually.

Reply:
nvm, I don't think so:

> 2.15.1 (stable) (Tue Dec 14 13:32:21 2021 +0100) on "android_x64"

Reply:
But it is a build string?

Reply:
The example above looks like more `rawDescription`.
The version is `2.15.1`, maybe `1` is the `build` but I prefer to not infer this since it's not a dedicated field under the `Platform` class.